src/uct/api/uct_def.h

```diff
@@ -18,6 +18,7 @@
 #define UCT_COMPONENT_NAME_MAX     16
 #define UCT_TL_NAME_MAX            10
 #define UCT_MD_NAME_MAX            16
+#define UCT_CM_NAME_MAX            16
```

Comment:
doesn't cm name duplicates component name? IIRC md_name can be different because it's bound to device.

Reply:
it does. so does md - https://github.com/openucx/ucx/blob/master/src/uct/base/uct_md.c#L153

Reply:
then why do we need it?